[framework] Remove redundant path segments (part 1)

nemrism opened the pull request:
Remove redundant path in relative links inside /framework/** files

Contributes to #10653

nemrism committed:
remove redundant path segments in root files

nemrism committed:
remove redundant path segments in app-domains/** files

nemrism committed:
remove redundant path segments in debug-trace-profile/** files

nemrism committed:
remove redundant path segments in deployment/** files

nemrism committed:
remove redundant path segments in interop/** files

nemrism committed:
remove redundant path segments in net-native/** files

nemrism committed:
remove redundant path segments in network-programming/** files

nemrism committed:
remove redundant path segments in performance/** files

nemrism committed:
remove redundant path segments in reflection-and-codedom/** files

nemrism committed:
remove redundant segments path in security/** files

nemrism committed:
remove redundant segments path in tools/** files

nemrism committed:
remove redundant segments path in ui-automation/** files

nemrism committed:
remove redundant segments path in windows-services/** files

mairaw commented:
I know these changes are super straightforward but let's try to keep these PRs with ~100 files. I'll review this one as-is though.

mairaw commented on docs/framework/additional-apis/index.md:
```diff
@@ -39,22 +39,22 @@ These APIs support the product infrastructure and are not intended/supported to
 ## See also
 
-- [The .NET Framework and Out-of-Band Releases](../../../docs/framework/get-started/the-net-framework-and-out-of-band-releases.md)
+- [The .NET Framework and Out-of-Band Releases](the-net-framework-and-out-of-band-releases.md)
```
I think this path needs to look like this:

```suggestion
- [The .NET Framework and Out-of-Band Releases](../get-started/the-net-framework-and-out-of-band-releases.md)
```

mairaw reviewed (changes requested):
Wonderful to see these changes. In addition to the broken link I've pointed out, I also have a few additional comments. I've reviewed 200/502 files so far.

mairaw commented on docs/framework/network-programming/using-an-asynchronous-server-socket.md:
```diff
@@ -264,7 +264,7 @@ public static void ReadCallback(IAsyncResult ar)
-- [Asynchronous Server Socket Example](../../../docs/framework/network-programming/asynchronous-server-socket-example.md)
+- [Using a Synchronous Server Socket](using-a-synchronous-server-socket.md)
+- [Asynchronous Server Socket Example](asynchronous-server-socket-example.md)
 - [Threading](../../../docs/standard/threading/index.md)
```
```suggestion
- [Threading](../../standard/threading/index.md)
```

mairaw commented on docs/framework/performance/etw-events.md:
```diff
@@ -14,14 +14,14 @@ Event tracing for Windows (ETW) is a high-performance, low-overhead, scalable tr
  In the .NET Framework, ETW event tracing is available for the common language runtime (CLR), the [Task Parallel Library](../../../docs/standard/parallel-programming/task-parallel-library-tpl.md), and [Parallel LINQ (PLINQ)](../../../docs/standard/parallel-programming/parallel-linq-plinq.md).  
```
```suggestion
 In the .NET Framework, ETW event tracing is available for the common language runtime (CLR), the [Task Parallel Library](../../standard/parallel-programming/task-parallel-library-tpl.md), and [Parallel LINQ (PLINQ)](../../standard/parallel-programming/parallel-linq-plinq.md).  
```

mairaw commented on docs/framework/performance/etw-events.md:
```diff
@@ -14,14 +14,14 @@ Event tracing for Windows (ETW) is a high-performance, low-overhead, scalable tr
 
-- [CLR ETW Events](../../../docs/framework/performance/clr-etw-events.md)
+- [CLR ETW Events](clr-etw-events.md)
 - [Task Parallel Library (TPL)](../../../docs/standard/parallel-programming/task-parallel-library-tpl.md)
```
```suggestion
- [Task Parallel Library (TPL)](../../standard/parallel-programming/task-parallel-library-tpl.md)
```

mairaw commented on docs/framework/performance/lazy-initialization.md:
```diff
@@ -151,4 +151,4 @@ ms.author: "ronpet"
 - [Managed Threading Basics](../../../docs/standard/threading/managed-threading-basics.md)
```
```suggestion
- [Managed Threading Basics](../../standard/threading/managed-threading-basics.md)
```

mairaw commented on docs/framework/performance/lazy-initialization.md:
```diff
@@ -151,4 +151,4 @@ ms.author: "ronpet"
 - [Managed Threading Basics](../../../docs/standard/threading/managed-threading-basics.md)
 - [Threads and Threading](../../../docs/standard/threading/threads-and-threading.md)
```
```suggestion
- [Threads and Threading](../../standard/threading/threads-and-threading.md)
```

mairaw commented on docs/framework/performance/lazy-initialization.md:
```diff
@@ -151,4 +151,4 @@ ms.author: "ronpet"
 - [Managed Threading Basics](../../../docs/standard/threading/managed-threading-basics.md)
 - [Threads and Threading](../../../docs/standard/threading/threads-and-threading.md)
 - [Task Parallel Library (TPL)](../../../docs/standard/parallel-programming/task-parallel-library-tpl.md)
```
```suggestion
- [Task Parallel Library (TPL)](../../standard/parallel-programming/task-parallel-library-tpl.md)
```

mairaw commented on docs/framework/reflection-and-codedom/dynamic-language-runtime-overview.md:
```diff
@@ -70,7 +70,7 @@ Examples of languages developed by using the DLR include the following:
  The following illustration shows the architecture of the dynamic language runtime.
 
- ![Dynamic Language Runtime Architecture Overview](../../../docs/framework/reflection-and-codedom/media/dlr-archoverview.png "DLR_ArchOverview")
+ ![Dynamic Language Runtime Architecture Overview](media/dlr-archoverview.png "DLR_ArchOverview")
```
```suggestion
 ![Dynamic Language Runtime Architecture Overview](./media/dlr-archoverview.png "DLR_ArchOverview")
```

mairaw commented on docs/framework/reflection-and-codedom/dynamically-loading-and-using-types.md:
```diff
@@ -97,5 +97,5 @@ End Module
 - <xref:System.Reflection.Assembly.Load%2A?displayProperty=nameWithType>
-- [Viewing Type Information](../../../docs/framework/reflection-and-codedom/viewing-type-information.md)
+- [Viewing Type Information](viewing-type-information.md)
 - [Type Conversion in the .NET Framework](../../../docs/standard/base-types/type-conversion.md)
```
```suggestion
- [Type Conversion in the .NET Framework](../../standard/base-types/type-conversion.md)
```

mairaw reviewed (commented):
A few more comments... 300/502 files reviewed.

mairaw commented on docs/framework/tools/winres-exe-windows-forms-resource-editor.md:
```diff
@@ -138,6 +138,6 @@ You should use the version of Winres.exe that was released with the .NET Framewo
-- [Resources in Desktop Apps](../../../docs/framework/resources/index.md)
+- [Tools](index.md)
+- [Resources in Desktop Apps](../resources/index.md)
 - [Globalization and Localization](../../../docs/standard/globalization-localization/index.md)
```
```suggestion
- [Globalization and Localization](../../standard/globalization-localization/index.md)
```

mairaw commented on docs/framework/ui-automation/access-embedded-objects-using-ui-automation.md:
```diff
@@ -21,7 +21,7 @@ ms.assetid: a5b513ec-7fa6-4460-869f-c18ff04f7cf2
  Embedded objects are considered children of the [!INCLUDE[TLA2#tla_uiautomation](../../../includes/tla2sharptla-uiautomation-md.md)] text provider. This allows them to be exposed through the same UI Automation tree structure as all other [!INCLUDE[TLA#tla_ui](../../../includes/tlasharptla-ui-md.md)] elements. Functionality, in turn, is exposed through the control patterns typically required by t ...
   
- ![Embedded objects in a text container.](../../../docs/framework/ui-automation/media/uia-textpattern-embeddedobjects.PNG "UIA_TextPattern_EmbeddedObjects")  
+ ![Embedded objects in a text container.](media/uia-textpattern-embeddedobjects.PNG "UIA_TextPattern_EmbeddedObjects")  
```
```suggestion
 ![Embedded objects in a text container.](./media/uia-textpattern-embeddedobjects.PNG "UIA_TextPattern_EmbeddedObjects")  
```

mairaw reviewed (commented):
More comments,,, 400/502

mairaw commented on docs/framework/ui-automation/implementing-the-ui-automation-table-control-pattern.md:
```diff
@@ -28,12 +28,12 @@ ms.assetid: 880cd85c-aa8c-4fb5-9369-45491d34bb78
+- See [Implementing the UI Automation Grid Control Pattern](implementing-the-ui-automation-grid-control-pattern.md) for related grid functionality.  
   
- ![Table with complex header items.](../../../docs/framework/ui-automation/media/uia-tablepattern-complex-column-headers.PNG "UIA_TablePattern_Complex_Column_Headers")  
+ ![Table with complex header items.](media/uia-tablepattern-complex-column-headers.PNG "UIA_TablePattern_Complex_Column_Headers")  
```
```suggestion
 ![Table with complex header items.](./media/uia-tablepattern-complex-column-headers.PNG "UIA_TablePattern_Complex_Column_Headers")  
```

mairaw commented on docs/framework/ui-automation/implementing-the-ui-automation-table-control-pattern.md:
```diff
@@ -28,12 +28,12 @@ ms.assetid: 880cd85c-aa8c-4fb5-9369-45491d34bb78
 Example of a Table with Complex Column Headers  
   
- ![Table with ambiguous RowOrColumnMajor property.](../../../docs/framework/ui-automation/media/uia-tablepattern-roworcolumnmajorproperty.PNG "UIA_TablePattern_RowOrColumnMajorProperty")  
+ ![Table with ambiguous RowOrColumnMajor property.](media/uia-tablepattern-roworcolumnmajorproperty.PNG "UIA_TablePattern_RowOrColumnMajorProperty")  
```
```suggestion
 ![Table with ambiguous RowOrColumnMajor property.](./media/uia-tablepattern-roworcolumnmajorproperty.PNG "UIA_TablePattern_RowOrColumnMajorProperty")  
```

mairaw commented on docs/framework/ui-automation/implementing-the-ui-automation-value-control-pattern.md:
```diff
@@ -13,15 +13,15 @@ ms.assetid: b0fcdd87-3add-4345-bca9-e891205e02ba
 - Controls such as <xref:System.Windows.Automation.ControlType.ListItem> and <xref:System.Windows.Automation.ControlType.TreeItem> must support <xref:System.Windows.Automation.ValuePattern> if the value of any of the items is editable, regardless of the current edit mode of the control. The parent control must also support <xref:System.Windows.Automation.ValuePattern> if the child items are edita ...
   
- ![Editable list item.](../../../docs/framework/ui-automation/media/uia-valuepattern-editable-listitem.PNG "UIA_ValuePattern_Editable_ListItem")  
+ ![Editable list item.](media/uia-valuepattern-editable-listitem.PNG "UIA_ValuePattern_Editable_ListItem")  
```
```suggestion
 ![Editable list item.](./media/uia-valuepattern-editable-listitem.PNG "UIA_ValuePattern_Editable_ListItem")  
```

mairaw commented on docs/framework/ui-automation/implementing-the-ui-automation-value-control-pattern.md:
```diff
@@ -32,7 +32,7 @@ Example of an Editable List Item
 - <xref:System.Windows.Automation.Provider.IValueProvider> must be implemented by controls such as the **Color Picker** selection control from [!INCLUDE[TLA#tla_word](../../../includes/tlasharptla-word-md.md)] (illustrated below), which supports string mapping between a color value (for example, "yellow") and an equivalent internal [!INCLUDE[TLA#tla_rgb](../../../includes/tlasharptla-rgb-md.md)]  ...
   
- ![Color picker with yellow highlighted.](../../../docs/framework/ui-automation/media/uia-valuepattern-colorpicker.png "UIA_ValuePattern_ColorPicker")  
+ ![Color picker with yellow highlighted.](media/uia-valuepattern-colorpicker.png "UIA_ValuePattern_ColorPicker")  
```
```suggestion
 ![Color picker with yellow highlighted.](./media/uia-valuepattern-colorpicker.png "UIA_ValuePattern_ColorPicker")  
```

mairaw commented on docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md:
```diff
@@ -20,10 +20,10 @@ ms.assetid: 93fdfbb9-0025-4b72-8ca0-0714adbb70d5
  Embedded objects are treated as individual elements within the control view of the [!INCLUDE[TLA2#tla_uiautomation](../../../includes/tla2sharptla-uiautomation-md.md)] tree. They are exposed as children of the text container so that they can be accessed through the same model as other controls in [!INCLUDE[TLA2#tla_uiautomation](../../../includes/tla2sharptla-uiautomation-md.md)].  
   
- ![Embedded Table with Image in a Text Container](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-example1.png "UIA_TextPattern_Embedded_Objects_Overview_Example1")  
+ ![Embedded Table with Image in a Text Container](media/uia-textpattern-embedded-objects-overview-example1.png "UIA_TextPattern_Embedded_Objects_Overview_Example1")  
```
```suggestion
 ![Embedded Table with Image in a Text Container](./media/uia-textpattern-embedded-objects-overview-example1.png "UIA_TextPattern_Embedded_Objects_Overview_Example1")  
```

mairaw commented on docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md:
```diff
@@ -20,10 +20,10 @@ ms.assetid: 93fdfbb9-0025-4b72-8ca0-0714adbb70d5
 Example of a Text Container with Table, Image, and Hyperlink Embedded Objects  
   
- ![Content view for the preceding example](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-example2.PNG "UIA_TextPattern_Embedded_Objects_Overview_Example2")  
+ ![Content view for the preceding example](media/uia-textpattern-embedded-objects-overview-example2.PNG "UIA_TextPattern_Embedded_Objects_Overview_Example2")  
```
```suggestion
 ![Content view for the preceding example](./media/uia-textpattern-embedded-objects-overview-example2.PNG "UIA_TextPattern_Embedded_Objects_Overview_Example2")  
```

mairaw commented on docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md:
```diff
@@ -32,7 +32,7 @@ Example of the Content View for a Portion of the Preceding Text Container
  The textual content (or inner text) of a text container and an embedded object, such as a hyperlink or table cell, is exposed as a single, continuous text stream in both the control view and the content view of the [!INCLUDE[TLA2#tla_uiautomation](../../../includes/tla2sharptla-uiautomation-md.md)] tree; object boundaries are ignored. If a UI Automation client is retrieving the text for the purp ...
   
- ![Text ranges spanned by embedded objects.](../../../docs/framework/ui-automation/media/uia-textpattern-embeddedobjecttextranges.png "UIA_TextPattern_EmbeddedObjectTextRanges")  
+ ![Text ranges spanned by embedded objects.](media/uia-textpattern-embeddedobjecttextranges.png "UIA_TextPattern_EmbeddedObjectTextRanges")  
```
```suggestion
 ![Text ranges spanned by embedded objects.](./media/uia-textpattern-embeddedobjecttextranges.png "UIA_TextPattern_EmbeddedObjectTextRanges")  
```

mairaw commented on docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md:
```diff
@@ -45,7 +45,7 @@ Example of a text stream with embedded objects and their range spans
 4. The range is then expanded from a degenerate range state by moving the <xref:System.Windows.Automation.Text.TextPatternRangeEndpoint.End> endpoint by one requested <xref:System.Windows.Automation.Text.TextUnit> boundary.  
   
- ![Range adjustments by Move & ExpandToEnclosingUnit](../../../docs/framework/ui-automation/media/uia-textpattern-moveandexpand-examples.png "UIA_TextPattern_MoveAndExpand_Examples")  
+ ![Range adjustments by Move & ExpandToEnclosingUnit](media/uia-textpattern-moveandexpand-examples.png "UIA_TextPattern_MoveAndExpand_Examples")  
```
```suggestion
 ![Range adjustments by Move & ExpandToEnclosingUnit](./media/uia-textpattern-moveandexpand-examples.png "UIA_TextPattern_MoveAndExpand_Examples")  
```

mairaw commented on docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md:
```diff
@@ -95,18 +95,18 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
  **Example 1 - A text range that contains an embedded image**  
   
- {The image ![Embedded Image Example](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text}.  
+ {The image ![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text}.  
```
```suggestion
 {The image ![Embedded Image Example](./media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text}.  
```

mairaw commented on docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md:
```diff
@@ -95,18 +95,18 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
  **Example 2 - A text range that partially spans the content of a text container. The text container has an embedded image that is not part of the text range.**  
   
- {The image} ![Embedded Image Example](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text.  
+ {The image} ![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text.  
```
```suggestion
 {The image} ![Embedded Image Example](./media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text.  
```

mairaw commented on docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md:
```diff
@@ -121,16 +121,16 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
-|![Embedded Image Example](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|  
-|![Embedded Image Example 2](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample2.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample2")|Y|  
-|![Embedded Image Example 3](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample3.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample3")<br /><br /> Image for Z|Z|  
+|![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|  
```
```suggestion
|![Embedded Image Example](./media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|  
```

mairaw commented on docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md:
```diff
@@ -121,16 +121,16 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
-|![Embedded Image Example 2](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample2.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample2")|Y|  
-|![Embedded Image Example 3](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample3.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample3")<br /><br /> Image for Z|Z|  
+|![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|  
+|![Embedded Image Example 2](media/uia-textpattern-embedded-objects-overview-imageexample2.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample2")|Y|  
```
```suggestion
|![Embedded Image Example 2](./media/uia-textpattern-embedded-objects-overview-imageexample2.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample2")|Y|  
```

mairaw commented on docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md:
```diff
@@ -121,16 +121,16 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
-|![Embedded Image Example 3](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample3.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample3")<br /><br /> Image for Z|Z|  
+|![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|  
+|![Embedded Image Example 2](media/uia-textpattern-embedded-objects-overview-imageexample2.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample2")|Y|  
+|![Embedded Image Example 3](media/uia-textpattern-embedded-objects-overview-imageexample3.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample3")<br /><br /> Image for Z|Z|  
```
```suggestion
|![Embedded Image Example 3](./media/uia-textpattern-embedded-objects-overview-imageexample3.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample3")<br /><br /> Image for Z|Z|  
```

mairaw commented on docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md:
```diff
@@ -121,16 +121,16 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
 |-------------------|------------|  
 |<xref:System.Windows.Automation.GridPattern.GetItem%2A> with parameters (0,0)|Returns the <xref:System.Windows.Automation.AutomationElement> representing the content of the table cell; in this case, the element is a text control.|  
-|<xref:System.Windows.Automation.TextPattern.RangeFromChild%2A> where <xref:System.Windows.Automation.AutomationElement> is the object returned by the previous `GetItem` method.|Returns the range that spans the image ![Embedded Image Example](../../../docs/framework/ui-automation/media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExamp ...
+|<xref:System.Windows.Automation.TextPattern.RangeFromChild%2A> where <xref:System.Windows.Automation.AutomationElement> is the object returned by the previous `GetItem` method.|Returns the range that spans the image ![Embedded Image Example](media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample").|  
```
```suggestion
|<xref:System.Windows.Automation.TextPattern.RangeFromChild%2A> where <xref:System.Windows.Automation.AutomationElement> is the object returned by the previous `GetItem` method.|Returns the range that spans the image ![Embedded Image Example](./media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample").|  
```

mairaw committed:
remove heading with _

mairaw commented on docs/framework/ui-automation/ui-automation-support-for-the-pane-control-type.md:
```diff
@@ -81,7 +81,7 @@ ms.assetid: 79761191-4449-4630-899c-9cbdb8867d3f
  The following image illustrates a control that implements the Pane control type.  
   
- ![Screenshot of applet window with two panes](../../../docs/framework/ui-automation/media/uiauto-pane.GIF "uiauto_pane")  
+ ![Screenshot of applet window with two panes](media/uiauto-pane.GIF "uiauto_pane")  
```
```suggestion
 ![Screenshot of applet window with two panes](./media/uiauto-pane.GIF "uiauto_pane")  
```

mairaw commented on docs/framework/ui-automation/ui-automation-support-for-the-splitbutton-control-type.md:
```diff
@@ -70,7 +70,7 @@ ms.assetid: 14b05ccf-bcd8-4045-9bae-f7679cd98711
  The following image illustrates a SplitButton control type in a data grid control.  
   
- ![Split button](../../../docs/framework/ui-automation/media/uiauto-splitbutton-detailed.gif "uiauto_splitbutton_detailed")  
+ ![Split button](media/uiauto-splitbutton-detailed.gif "uiauto_splitbutton_detailed")  
```
```suggestion
 ![Split button](./media/uiauto-splitbutton-detailed.gif "uiauto_splitbutton_detailed")  
```

mairaw commented on docs/framework/ui-automation/ui-automation-textpattern-overview.md:
```diff
@@ -124,12 +124,12 @@ A formatting characteristic of a text range (for example, <xref:System.Windows.A
 The absolute <xref:System.Windows.Automation.Text.TextPatternRangeEndpoint.Start> or <xref:System.Windows.Automation.Text.TextPatternRangeEndpoint.End> point of a text range within a text container.
 
-![TextPatternRangeEndpoints &#40;start and end&#41;.](../../../docs/framework/ui-automation/media/uia-textpattern-endpoints.PNG "UIA_TextPattern_Endpoints")
+![TextPatternRangeEndpoints &#40;start and end&#41;.](media/uia-textpattern-endpoints.PNG "UIA_TextPattern_Endpoints")
```
```suggestion
![TextPatternRangeEndpoints &#40;start and end&#41;.](./media/uia-textpattern-endpoints.PNG "UIA_TextPattern_Endpoints")
```

mairaw committed:
remove heading with _

mairaw commented on docs/framework/windows-services/how-to-create-windows-services.md:
```diff
@@ -26,11 +26,11 @@ When you create a service, you can use a Visual Studio project template called *
 2. In the **Properties** window, set the <xref:System.ServiceProcess.ServiceBase.ServiceName%2A> property for your service.  
   
-     ![Set the ServiceName property.](../../../docs/framework/windows-services/media/windowsservice-servicename.PNG "WindowsService_ServiceName")  
+     ![Set the ServiceName property.](media/windowsservice-servicename.PNG "WindowsService_ServiceName")  
```
```suggestion
     ![Set the ServiceName property.](./media/windowsservice-servicename.PNG "WindowsService_ServiceName")  
```

mairaw reviewed (commented):
Finished the review now. A couple more comments...

Also, apologies for including two commits here that were supposed to be done on a separate PR I'm working on. It's fine to leave them here if they don't cause any issues.

nemrism committed:
Update docs/framework/additional-apis/index.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/develop-web-apps-with-aspnet.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/reflection-and-codedom/dynamic-language-runtime-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/windows-services/how-to-create-windows-services.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/ui-automation-textpattern-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/ui-automation-support-for-the-splitbutton-control-type.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/ui-automation-support-for-the-pane-control-type.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/implementing-the-ui-automation-value-control-pattern.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/implementing-the-ui-automation-value-control-pattern.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/implementing-the-ui-automation-table-control-pattern.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/implementing-the-ui-automation-table-control-pattern.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/ui-automation/access-embedded-objects-using-ui-automation.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/security/wsfederation-authentication-module-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/install/guide-for-developers.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/security/claims-based-identity-model.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism committed:
Update docs/framework/security/wsfederation-authentication-module-overview.md

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

nemrism commented:
I applied all suggested fixes about "incorrect" includes of `media` files.

I've changed all links to `/standard` section in the following PR: #13925 as this replacement work is quite huge even if I split by folders...
Once it will be merged I could rebase this branch in order to "facilitate" the final review.

nemrism committed:
fix incorrect media includes

mairaw commented:
@nemrism can you fix the merge conflicts?

nemrism commented:
Yes, I'll push that asap

nemrism committed:
Merge remote-tracking branch 'origin/master' into redundant-path-framework-1

nemrism commented:
Merged, resolved and ready for your review @mairaw

mairaw commented:
Thanks @nemrism. @rpetrusha I'm afraid this will cause new merge conflicts for #13851. Should we hold off until that one is merged?

nemrism committed:
Merge branch 'master' into redundant-path-framework-1

nemrism committed:
Merge branch 'master' into redundant-path-framework-1

nemrism committed:
Merge branch 'master' into redundant-path-framework-1

nemrism commented:
cc @mairaw / @rpetrusha 

Just tested the [v-thepet/move2](https://github.com/v-thepet/docs/tree/move2) branch against this one. It is indeed the most impacted one by merge conflicts but it is quite easy to resolve them.

Let me know when the #13851 PR will be merged, I'll handle the conflicts.

I'll continue to work on the #10653 resolution, it seems that other section are less impacted, automatic merge works fine on most of other related PR.

nemrism committed:
Merge branch 'master' into redundant-path-framework-1

nemrism committed:
Merge branch 'master' into redundant-path-framework-1

nemrism committed:
Merge branch 'master' into redundant-path-framework-1

nemrism committed:
Merge branch 'master' into redundant-path-framework-1

mairaw commented:
That PR was now merged. Can you please rebase your PR? Thanks!

nemrism commented:
Yes of course!
I'll prepare that and tell you when this one is ready for final review.

nemrism committed:
Merge branch 'master' into redundant-path-framework-1

nemrism commented:
All done, ready for final review @mairaw

mairaw reviewed (approved):
Changes look good! Thank you @nemrism!
